src/main/kotlin/racing_car/domain/car/RandomMovingEngine.kt

```diff
@@ -0,0 +1,17 @@
+package racing_car.domain.car
+
+class RandomMovingEngine(
+    private val minRandomValue: Int = 0,
+    private val maxRandomValue: Int = 9,
+    private val moveThreshold: Int = 4,
+    private val defaultMoveDistance: Int = 1
+) :
+
+    Engine {
```

Comment:
사소한 부분이긴 하지만 [컨벤션](https://kotlinlang.org/docs/coding-conventions.html#class-headers)은 최대한 지켜서 작성해주세요🥲
```suggestion
) : Engine {
```

Reply:
헉 klint가 다 잡아주는 건 아니군요!